scripts/ci/gke.sh

```diff
@@ -316,13 +316,16 @@ refresh_gke_token() {
 
 teardown_gke_cluster() {
     local canceled="${1:-false}"
+    local byodb="${BYODB_TEST:-false}"
 
     info "Tearing down the GKE cluster: ${CLUSTER_NAME:-}, canceled: ${canceled}"
 
     require_environment "CLUSTER_NAME"
     require_executable "gcloud"
 
-    if [[ "${canceled}" == "false" ]]; then
+    if [[ "${canceled}" == "false" ]] &&
+       [[ "${byodb}" == "false" ]]
+    then
```

Comment:
Someone smarter than me may have a better idea here.  This was causing the teardown to timeout at 5 minutes.  I even extended it to 10 minutes and it was still timing out.  I'm assuming it is because the teardown of the database was making things take too long, but I couldn't figure out why.  I also wasn't quite sure of the point of tearing down before destroying the cluster anyway.  So I skip cleaning up deployment if byodb tests.